asf_search/ASFProduct.py

```diff
@@ -11,6 +11,7 @@ class ASFProduct:
     def __init__(self, args: dict):
         self.properties = args['properties']
         self.geometry = args['geometry']
+        self.baseline = self.properties.pop('baseline')
```

Comment:
If we go with your original notion of altering SearchAPI's asf_search format to have baseline as its own attribute adjacent to geometry and properties, we could normalize this line to
self.baseline = args['baseline']
and we wouldn't need to be "tidying up" properties by way of pop().

Previously, I may have unintentionally suggested leaving SearchAPI alone, but I didn't mean to roll back those changes, I think it was a good idea to handle this optimally at both ends.